README.rst

```diff
@@ -434,6 +434,11 @@ Build and publish the new version:
 
 Changelog
 =========
+2.7
+---
+
+* ``Sequence.last`` field changed from ``PositiveIntegerField`` to ``PositiveBigIntegerField``
```

Comment:
I think we should have a few more words covering that this requires a migration, and thatit may take a while on projects with many sequences. I'm not sure there's any general purpose advice we can give to users but we should highlight the potential for locking during deployment.

Reply:
Added detailed info. Did my best :slightly_smiling_face: